### DISCUSSION TITLE
Waiting for TimePicker And Date-Range-Picker.Very basic form compnent.

### DISCUSSION THREAD

**Discussion by velly** [vuetifyjs/vuetify#17788] [General]
FYI.
Create a thead to tracing and dicussing on the TimePicker's development plan.

**Comment by harkus** [comment]
I think this is the last component that I used in Vuetify 2 that I'm waiting on. I'm looking forward to it sooner than later, I hope :)

**Comment by velly** [comment]
Update: This component is planed after V3.4.

**Reply by harkus** [reply]
Good to know, thank you :)

**Reply by velly** [reply]
Up~~

**Comment by velly** [comment]
If If the calendar cannot support range selection, then the user will be clicked to deadth for a long period?![From 3.4]
